src/main/anovos/feature_recommender/featrec_init.py

```diff
@@ -1,6 +1,70 @@
-from anovos.feature_recommender.feature_exploration import *
+from sentence_transformers import SentenceTransformer
 from re import finditer
+import pandas as pd
+import numpy as np
 import copy
+import warnings
+
+
+warnings.filterwarnings("ignore")
```

Comment:
```suggestion
```

Please don't hide warnings in library code. If a user doesn't want to see warnings, they can suppress them further up in their own code.